public/1.html

```diff
@@ -5,14 +5,72 @@
     <title>TODOS</title>
 </head>
 <body>
+    <header>
+        <h1 class="title">Details Page</h1>
+    </header>
     <main>
-        <h1 class="title">Details</h1>
         <section class="listing-detail">
          <p>Todo #1</p>
         </section>
         <section>
           <a href="/todo" rel="collection">Back to Listings</a>
         </section>
     </main>
+    <footer>
+        &copy 2019 @angieg0nzalez | 8th Light Apprenticeship
+    </footer>
 </body>
-</html>
\ No newline at end of file
+</html>
+
+<style>
+    body {
+        align-items: center;
+        display: flex;
+        flex-direction: column;
+        font-family: Courier New;
+        margin: 0;
+        min-height: 100vh;
+    }
+
+    a {
+        text-decoration: none;
+    }
+
+    .todo-list {
+        width: 40rem;
+    }
+
+    .title {
+        align-self: center;
+        color: rebeccapurple;
+        font-size: 2.5rem;
+    }
+
+    ul {
```

Comment:
I like the use of semantic HTML overall.  There are some tags here I have been meaning to use more of, myself.  I also like how `.title` and `.todo-list` are more semantic instead of using `h1` et al.  You might apply that idea for `ul` and `li` as well -- maybe call it something like `.todo-list` and `.todo-item`?